examples/src/main/java/alluxio/examples/MultiMount.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the “License”). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.examples;
+
+import alluxio.AlluxioURI;
+import alluxio.client.WriteType;
+import alluxio.client.file.FileInStream;
+import alluxio.client.file.FileOutStream;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.options.CreateFileOptions;
+
+import org.apache.commons.io.IOUtils;
+
+/**
+ * Example program that demonstrates Alluxio's ability to read and write data across different
+ * types of storage systems. In particular, this example reads data from S3 and writes data to HDFS.
+ *
+ * NOTE: To run this example, you must replace the "hdfs://localhost:9000/" URL with a
+ * URL of a valid HDFS cluster. Also, you need to set the fs.s3n.awsAccessKeyId and
+ * fs.s3.awsSecretAccessKey VM properties to a valid AWS access key ID and aws secret access key
+ * respectively in order to access the S3 bucket the data is read from.
+ */
+public final class MultiMount {
+
+  /**
+   * Entry point for the {@link MultiMount} program.
+   *
+   * @param args command-line arguments
+   */
+  public static void main(String []args) {
+    FileSystem fileSystem = FileSystem.Factory.get();
+    try {
+      AlluxioURI mntPath = new AlluxioURI("/mnt");
+      AlluxioURI s3Path = new AlluxioURI("/mnt/s3");
+      AlluxioURI hdfsPath = new AlluxioURI("/mnt/hdfs");
+      // Make sure that the directory under which mount points are created exists.
+      if (!fileSystem.exists(mntPath)) {
+        fileSystem.createDirectory(mntPath);
+      }
+      // Make sure that the S3 mount point does not exist.
+      if (fileSystem.exists(s3Path)) {
+        fileSystem.unmount(s3Path);
+      }
+      // Make sure that the HDFS mount point does not exist.
+      if (fileSystem.exists(hdfsPath)) {
+        fileSystem.unmount(hdfsPath);
+      }
+      // Create the S3 and HDFS mount points.
+      fileSystem.mount(new AlluxioURI("/mnt/s3"), new AlluxioURI("s3n://alluxio-demo/"));
```

Comment:
should we have some console messages to illustrate what's going on at each steps?